composer.json

```diff
@@ -50,7 +50,7 @@
     }
   ],
   "require": {
-    "php": ">=8.0",
+    "php": ">=8.1",
```

Comment:
This could potentially cause merge conflicts. If Bedrock requires PHP 8.1+, and since we require a couple of Bedrock dependencies, we should pick up their minimum PHP version automatically.

If it did turn out to be important to bump this for some reason, we could do it [at the same time we adjust the platform.php version](https://github.com/pantheon-systems/wordpress-composer-managed/blob/default/upstream-configuration/scripts/ComposerScripts.php#L63), or at least in the same vicinity.

Reply:
```suggestion
    "php": ">=8.0",
```